api/src/main/java/org/apache/iceberg/io/FileIO.java

```diff
@@ -68,6 +69,17 @@ default void deleteFile(OutputFile file) {
   default void initialize(Map<String, String> properties) {
   }
 
+  /**
+   * Delete the files at the given path.
+   *
+   * @throws FileIODeletionException If failures are encountered when deleting the given files
```

Comment:
This needs to document `pathsToDelete` as well.

Reply:
Updated